tracex_project/tracex/logic/utils.py

```diff
@@ -11,6 +11,10 @@
 import pandas as pd
 import pm4py
 
+
+from pm4py.objects.conversion.log import converter as log_converter
+
+
```

Comment:
I think, one line before and after the import should be enough. But taking into account, that `pm4py` is already imported, the import might be unnecessary or if we need it, it would be fitting better right next to the `pm4py` import